demo/index.html

```diff
@@ -142,6 +142,49 @@
     }());
   </script>
 
+  <p>Here's a pie chart listening for `onmouseover`:</p>
+
+  <style>
+    #mouseover-demo {
+      position: relative;
+      height: 300px;
+    }
+
+    #mouseover-chart {
+      float: left;
+    }
+
+    #mouseover-display {
+      display: inline-block;
+      position: relative;
+      top: 50%;
+    }
+  </style>
+  <div id="mouseover-demo">
+    <google-chart
+      id="mouseover-chart"
+      type="pie"
+      options='{"title": "Distribution of days in 2001H1"}'
+      cols='[{"label": "Month", "type": "string"},{"label": "Days", "type": "number"}]'
+      rows='[ ["Jan", 31],["Feb", 28],["Mar", 31],["Apr", 30],["May", 31],["Jun", 30] ]'
+      events='["onmouseover"]'>
```

Comment:
I wonder if there's a better way to do this than have users provide the events array. One thing that the maps component does is register a set of events. For example, adding the `click-events` attr will forward the maps click events to element code. 

https://github.com/GoogleWebComponents/google-map/blob/master/google-map.html#L325

How many events does charts have?